dds/DCPS/DiscoveryBase.h

```diff
@@ -100,8 +100,8 @@ namespace OpenDDS {
         , reader_done_(false), writer_done_(false), cnd_(mtx_)
         , interval_(TimeDuration(0))
         , status_(0)
-        , tid_(0)
       {
+        tid_ = 0;
```

Comment:
Why was this changed?

Reply:
The initializer caused an error in the conformance test build: DCPS/DiscoveryBase.h:103: error: no matching function for call to 'pthread_t::pthread_t(int)'